samples/VirtualizationDemo/ViewModels/MainWindowViewModel.cs

```diff
@@ -18,7 +18,7 @@ internal class MainWindowViewModel : ReactiveObject
         private int _itemCount = 200;
         private string _newItemString = "New Item";
         private int _newItemIndex;
-        private IReactiveList<ItemViewModel> _items;
+        private AvaloniaList<ItemViewModel> _items;
```

Comment:
Probably worth using `ObservableCollectionExtended` from DynamicData here and there to reduce coupling with the GUI framework being used, as [DynamicData is actually the replacement](https://reactiveui.net/docs/handbook/collections/) for `ReactiveList<T>`. Although this shouldn't stop the PR from being merged as there are other issues with coupling in this file, and the upgrade to ReactiveUI 10 ideally should be done before next Avalonia release NuGet.